tree/dataframe/inc/ROOT/RDF/RInterface.hxx

```diff
@@ -2227,15 +2313,14 @@ public:
    /// RDataFrame df(10);
```

Comment:
must update the method description above, e.g. it shouldn't point users to `DescribeDataset`

Reply:
fixed in the latest commit